conans/tools.py

```diff
@@ -10,6 +10,7 @@
 import subprocess
 import sys
 
+import io
```

Comment:
It seems that the io module is available for both Python 2.7 and 3.x.
Anyway, I removed it.